OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -64,13 +64,23 @@ protected static Actor FindDefenselessTarget(Squad owner)
 		{
 			var map = owner.World.Map;
 			var dangerRadius = owner.SquadManager.Info.DangerScanRadius;
+
+			// DangerScanRadius of zero or less causes infinite loop, so use sensible fallback.
```

Comment:
This should be done in `SquadManagerBotModuleInfo` by using `IRulesetLoaded` and throwing an exception if an invalid value is given.